tests/selenium/test_basics.py

```diff
@@ -37,3 +46,25 @@ def test_perfherder_main(initial_data, live_server, selenium):
     WebDriverWait(selenium, 10).until(
         EC.presence_of_element_located((By.ID, 'performance-test-chooser'))
     )
+
+
+def test_treeherder_single_commit_titles(initial_data, live_server, selenium):
+    '''
+    This tests that page titles are correct
+    '''
+    push = Push.objects.create(repository=Repository.objects.get(name='mozilla-central'),
+                               revision="1234abcd",
```

Comment:
40 character SHA please (will mean fewer places to change when we make this mandatory)